tests/components/azure_devops/test_config_flow.py

```diff
@@ -12,6 +13,8 @@
 )
 from homeassistant.core import HomeAssistant
 
+from tests.common import mock_coro
```

Comment:
`mock_coro` is legacy. Please import all mocks and patch from `tests.async_mock`. The `patch` there will detect when it's patching coroutines and use `AsyncMock` automatically. We can also use the latter explicitly when needed.

Reply:
Aha that's simpler. Cheers :+1: